src/shared/clients/ecsClient.ts

```diff
@@ -84,12 +84,28 @@ export class DefaultEcsClient {
             return listTasksResponse.taskArns ?? []
         } catch (error) {
             getLogger().error(
-                `ecs: Failed to get tasks for Cluster "${cluster}" and Service "${serviceName}": ${error} `
+                `ecs: Failed to get tasks for Cluster "${cluster}" and Service "${serviceName}": ${error}`
             )
             throw error
         }
     }
 
+    public async updateService(cluster: string, serviceName: string, enable: boolean): Promise<void> {
+        const sdkClient = await this.createSdkClient()
+
+        const params: ECS.UpdateServiceRequest = {
+            cluster,
+            service: serviceName,
+            enableExecuteCommand: enable,
+            forceNewDeployment: true,
+        }
+        try {
+            sdkClient.updateService(params).promise()
+        } catch (error) {
+            getLogger().error(`ecs: Failed to update service: ${error} `)
```

Comment:
This log statement doesn't add much value. Better to just let the error bubble up and let something else handle it with a more specific message.